app/views/kitchens/show.html.haml

```diff
@@ -1,10 +1,27 @@
-%p
-  %strong Floor:
-  = @kitchen.floor
-- @kitchen.appliances.each do |appliance|
-  %div
-    = link_to appliance.name, appliance_path(appliance)
-.mt-4
-  = link_to 'Edit', edit_kitchen_path(@kitchen), class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-  = link_to 'Back', kitchens_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
+.flex.flex-wrap
+  .flex-none.pr-2
+    %h2.text-2xl.md:text-4xl.w-full
+      %strong= "#{@kitchen.floor}. szinti konyha"
+  .flex-auto.mb-4
+    .text-right.py-2
+      = link_to 'Módosítás', edit_kitchen_path(@kitchen), class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
+
+.mb-4.text-lg.md:text-xl
+  A szint eszközei:
 
+.w-full
+  - @kitchen.appliances.each do |appliance|
+    - bg_color = appliance.status == "operational" ? "bg-green-600" : "bg-red-600"
+    = link_to appliance, class: "flex flex-wrap items-center cursor-pointer my-1 text-white shadow hover:text-gray-700 hover:bg-white rounded " + bg_color do
+      .flex-auto.p-3.break-words
+        %span.md:text-lg
+          %strong
+            = appliance.name
+        - if appliance.category == "extra"
+          %span.text-xs.font-medium.bg-yellow-200.text-gray-800.py-1.px-2.rounded.align-top
+            extra!
+      .flex-auto.p-3.text-right
+        = appliance.status
+
+.mt-10
+  = link_to 'Back', kitchens_path, class: "bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
```

Comment:
If the edit button and the 'szint eszközei' label, etc... are in Hungarian, the back button should be too.

Reply:
This should be fixed after #21, no need to change it twice.

Reply:
Do I fix it back? Or could you rebase your branch later if this PR gets merged?